README.md

```diff
@@ -35,8 +35,33 @@ This will introduce a few factories, namely you can retrieve the following objec
 
 ### Configure Pikkuleipa
 
-For Pikkuleipa to function, it needs a few configuration variables. Copy the file `doc/example-config.php` and adjust the
-values as needed.
+For Pikkuleipa to function, it needs a few configuration variables and RSA key pair for signing JWTs. 
```

Comment:
Technically you are not required to use RSA keys. The default SHA256 signing mechanism uses the same key for signature and verification. Only the RSA mechanism (which you actually omitted from the config example) requires that.